src/parser/name_resolution.c

```diff
@@ -10234,3 +10234,80 @@ pt_bind_name_to_spec (PARSER_CONTEXT * parser, PT_NODE * node, void *arg, int *c
   node->info.name.meta_class = PT_NORMAL;	// so far, only normals are used.
   return node;
 }
+
+static void
+pt_resolve_dblink_server_name (PARSER_CONTEXT * parser, PT_NODE * node)
+{
+  PT_NODE *val[3];
+  DB_VALUE values[3];
+  PT_DBLINK_INFO *dblink_table = &node->info.dblink_table;
+  int i;
+
+  assert (dblink_table->conn && dblink_table->conn->node_type == PT_NAME);
+
+  db_make_null (&(values[0]));
+  db_make_null (&(values[1]));
+  db_make_null (&(values[2]));
+  i = get_dblink_info_from_dbserver (parser, dblink_table->conn->info.name.original, values);
```

Comment:
variable "error" looks better than "i"

Reply:
It's a reuse of a variable, I'll  fix it for readability.